src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -12,7 +12,10 @@
       "taghelper",
       "aspnetcore",
       "taghelpers"
-    ]
+    ],
+    "files": {
+      "include": [ "buildCrossTargeting/*", "build/*" ]
```

Comment:
Ah I see. I'd still need the `buildCrossTargeting` bit to work for cases of `dotnet msbuild /t:ResolveRazorTargetFramework` since there's no `TargetFramework` specified. Will add `netstandard1.0` to allow for future flexibility if we need it.